pkg/component/machinecontrollermanager/machine_controller_manager.go

```diff
@@ -213,7 +213,7 @@ func (m *machineControllerManager) Deploy(ctx context.Context) error {
 						"--safety-up=2",
 						"--safety-down=1",
 						"--target-kubeconfig=" + gardenerutils.PathGenericKubeconfig,
-						"--v=4",
+						"--v=2",
```

Comment:
```suggestion
						"--v=3",
```
I checked that `--v=3` doesn't repeat this leader election message..

Reply:
Yes, But 3 is still high, Isn't it?

Reply:
Updated it 3. PTAL.